<ui version="4.0" >
 <class>qutimSettingsClass</class>
 <widget class="QDialog" name="qutimSettingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../qutim.qrc" >
    <normaloff>:/icons/crystal_project/settings.png</normaloff>:/icons/crystal_project/settings.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QComboBox" name="accountBox" >
     <property name="minimumSize" >
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QStackedWidget" name="settingsStack" >
     <property name="minimumSize" >
      <size>
       <width>330</width>
       <height>16</height>
      </size>
     </property>
     <property name="currentIndex" >
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QTreeWidget" name="settingsTree" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alternatingRowColors" >
      <bool>false</bool>
     </property>
     <property name="verticalScrollMode" >
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="horizontalScrollMode" >
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="animated" >
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>71</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="icon" >
        <iconset resource="../qutim.qrc" >
         <normaloff>:/icons/crystal_project/apply.png</normaloff>:/icons/crystal_project/apply.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Apply</string>
       </property>
       <property name="icon" >
        <iconset resource="../qutim.qrc" >
         <normaloff>:/icons/crystal_project/apply.png</normaloff>:/icons/crystal_project/apply.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="../qutim.qrc" >
         <normaloff>:/icons/crystal_project/cancel.png</normaloff>:/icons/crystal_project/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../qutim.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>qutimSettingsClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>495</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel" >
     <x>158</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
